keps/sig-storage/3333-reconcile-default-storage-class/kep.yaml

```diff
@@ -15,13 +15,14 @@ approvers:
 see-also:
 replaces:
 
-# The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+prr-approvers:
+  - "@deads2k"
+  - "@wojtek-t"
```

Comment:
Please remove this section.

And readd back stage: beta - lack of it is failing validation.